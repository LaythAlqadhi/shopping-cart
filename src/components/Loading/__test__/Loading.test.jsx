import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import userEvent from '@testing-library/user-event';
import Loading from '../Loading';

describe('Loading component', () => {
  it('renders loading element correctly', () => {
    const { container } = render(<Loading />)

    expect(container).toMatchSnapshot();
  })
})