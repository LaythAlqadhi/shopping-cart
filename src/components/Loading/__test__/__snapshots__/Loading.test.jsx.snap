// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Loading component > renders loading element correctly 1`] = `
<div>
  <div
    class="_loading_55dd7a"
  />
</div>
`;
